src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -136,6 +165,22 @@ def _sanitize_parameters(self, **kwargs):
         # No parameters on this pipeline right now
         return {}, {}, {}
 
+    def run_single(self, inputs, preprocess_params, forward_params, postprocess_params):
+        model_inputs = self.preprocess(inputs, **preprocess_params)
+        if self.chunking:
+            model_outputs = []
+            for chunk in model_inputs:
+                # run inference on padded chunks, but collect only the upadded outputs
+                chunk_output = self.forward(chunk, **forward_params)
+                model_outputs.append(
+                    chunk_output[self.chunk_output_padding_len : self.chunk_output_padding_len + self.chunk_output_len]
+                )
+            model_outputs = torch.cat(model_outputs)
```

Comment:
Ok for me now since we only allow PyTorch for the moment. Maybe we'll have to do some kind of if-else further down the road